CMakeLists.txt

```diff
@@ -21,19 +21,29 @@ ign_configure_project(VERSION_SUFFIX)
 # Set project-specific options
 #============================================================================
 
-# ignition-utils currently has no options that are unique to it
+option(
+  IGN_UTILS_VENDOR_CLI11
+  "If true, use the vendored version of CLI11, otherwise use an external one"
+  true)
 
 #============================================================================
 # Search for project-specific dependencies
 #============================================================================
 
-# ignition-utils shouldn't have any dependencies besides ign-cmake
+if(IGN_UTILS_VENDOR_CLI11)
+  set(IGN_UTILS_COMPONENTS "cli")
+else()
+  # PKGCONFIG_IGNORE can be removed once pkg-config support in CLI11 (see
+  # https://github.com/CLIUtils/CLI11/pull/523) is released
+  ign_find_package(CLI11 REQUIRED_BY cli PKGCONFIG_IGNORE)
```

Comment:
I can't tell if the `REQUIRED_BY` argument should be `cli` or `external-cli`

Reply:
This comment should no be solved as `external-cli` is not present anymore in the main CMakeLists.txt .